FWCore/Concurrency/interface/WaitingTaskWithArenaHolder.h

```diff
@@ -60,6 +60,12 @@ namespace edm {
     // the task. doneWaiting can be called from a non-TBB thread.
     void doneWaiting(std::exception_ptr iExcept);
 
+    // Use in the case where you need to inform the parent task of a
+    // failure before some other child task which may be run later
+    // reports a different, but related failure. You must later call
+    // doneWaiting in the same thread passing the same exceptoin.
```

Comment:
```suggestion
    // doneWaiting in the same thread passing the same exception.
```

Reply:
Thanks, fixed (also the original where I copied the comment from)